Sources/PackageModel/Destination.swift

```diff
@@ -166,12 +186,22 @@ public struct Destination: Equatable {
         hostTriple: Triple? = nil,
         targetTriple: Triple? = nil,
         sdkRootDir: AbsolutePath?,
-        toolset: Toolset
+        toolset: Toolset,
+        swiftResourcesPath: AbsolutePath? = nil,
+        swiftStaticResourcesPath: AbsolutePath? = nil,
+        includeSearchPaths: [AbsolutePath]? = nil,
+        librarySearchPaths: [AbsolutePath]? = nil,
+        toolsetPaths: [AbsolutePath]? = nil
```

Comment:
consider a Location struct to wrap all these paths?

Reply:
WDYT of `PathsConfiguration` as a name?